server/bleep/build.rs

```diff
@@ -21,9 +21,10 @@ fn main() {
 fn set_index_version() {
     use std::fs::{read_dir, read_to_string};
 
-    let model_directories = &["src/indexes", "src/intelligence/scope_resolution"];
+    let model_directories = &["src/intelligence/scope_resolution"];
     let model_files = &[
-        "src/indexes.rs",
+        "src/semantic/schema.rs",
+        "src/indexes/schema.rs",
```

Comment:
changes to, say `File::new` (currently in `src/indexes/file.rs`), such as modifying the index options of a field, could also require a re-index.